pkg/rootless/rootless_linux.go

```diff
@@ -465,9 +465,8 @@ func TryJoinFromFilePaths(pausePidPath string, needNewNamespace bool, paths []st
 		return BecomeRootInUserNS(pausePidPath)
 	}
 
-	var lastErr error
+	lastErr := error(unix.ESRCH)
```

Comment:
Won't this caluse lastErr to always be not nil, causing function to return an error?

Reply:
pushed a new version